apps/dashboard/src/lib/wallet/nfts/types.ts

```diff
@@ -51,6 +50,22 @@ import {
   zoraSepolia,
 } from "thirdweb/chains";
 
+type NFT = {
```

Comment:
where do we even still need this?

Reply:
in about 2 dozen places sadly

Reply:
for the NFT api responses (alchemy / moralis / ...) to be specific

Reply:
okay time to remove with chainsaw shortly